hr_timesheet_auto_input/wizards/wizard_timesheet_auto_input.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 OpenSynergy Indonesia
+# Copyright 2020 PT. Simetri Sinergi Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields, api
+
+
+class WizardTimesheetAutoInput(models.TransientModel):
+    _name = "hr_timesheet_sheet.sheet.auto_input"
+    _description = "Timesheet Auto Input"
+
+    timesheet_account_id = fields.Many2one(
+        string="Timesheet Account",
+        comodel_name="account.analytic.account",
+        domain=[("type", "<>", "view")],
+    )
+    task_id = fields.Many2one(
+        string="Task",
+        comodel_name="project.task",
+    )
+    product_id = fields.Many2one(
+        string="Product",
+        comodel_name="product.product",
+        domain=[("type", "=", "service")],
+    )
+    description = fields.Text(
+        string="Description",
+    )
+
+    @api.multi
+    @api.onchange(
+        "timesheet_account_id",
+    )
+    def onchange_task_id(self):
+        self.task_id = False
+        if self.timesheet_account_id:
+            obj_project = self.env["project.project"]
+            criteria = [
+                ("analytic_account_id", "=", self.timesheet_account_id.id)
+            ]
+            project_ids = obj_project.search(criteria)
+            if project_ids:
+                task_id = project_ids.tasks.ids
+            return {"domain": {"task_id": [("id", "in", task_id)]}}
```

Comment:
```suggestion
    @api.multi
    @api.onchange(
        "timesheet_account_id",
    )
    def onchange_task_id(self):
        self.task_id = False
```

Onchangenya cukup itu aja mike. Domainnya nanti tinggal dikontrol dari domain di view